controllers/workspaceitem_controller.go

```diff
@@ -62,27 +63,30 @@ func (r *WorkspaceItemReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 	if instance.IsBeingDeleted() {
 		r.Log.Info(fmt.Sprintf("HandleFinalizer for %v", req.NamespacedName))
 		if err := r.handleFinalizer(instance); err != nil {
+			r.Recorder.Event(instance, corev1.EventTypeWarning, "deleting finalizer", fmt.Sprintf("Failed to delete finalizer: %s", err))
 			return ctrl.Result{}, fmt.Errorf("error when handling finalizer: %v", err)
 		}
-		r.Recorder.Event(instance, "Normal", "Deleted", "Object finalizer is deleted")
+		r.Recorder.Event(instance, corev1.EventTypeNormal, "Deleted finalizer", "Object finalizer is deleted")
```

Comment:
Can you please update it to `"Deleted", "Object finalizer is deleted"` to following the same convention?